web/package.json

```diff
@@ -10,9 +10,11 @@
     "@portabletext/react": "^2.0.0",
     "@sanity/image-url": "^1.0.1",
     "groq": "^2.33.2",
+    "hamburger-react": "^2.5.0",
     "next": "latest",
     "next-transpile-modules": "^9.1.0",
     "react": "^18.2.0",
+    "react-burger-menu": "^3.0.8",
```

Comment:
Whu did you pull in 2 hamburger libraries? are you using them both?

Reply:
im using none of them i was going to use them but then i opted for creating my own from scratch